services/ux-backend/handlers/onboardingtokens/handler.go

```diff
@@ -52,3 +59,26 @@ func handleUnsupportedMethod(w http.ResponseWriter, r *http.Request) {
 		klog.Errorf("failed write data to response writer: %v", err)
 	}
 }
+
+func getQuantity(r *http.Request) (*resource.Quantity, error) {
+	var quota = struct {
+		Quota string `json:"quota"`
+	}{}
+	err := json.NewDecoder(r.Body).Decode(&quota)
+	if err != nil && err.Error() != "EOF" {
+		return nil, err
+	}
+
+	// When length is 0 that means either request body is empty or
+	// quota parameter is not passed in that case quota is considerred
+	// to be unlimited
```

Comment:
no, we already established that empty body represents unlimited quota and better not to club that w/ empty value for quota.

btw, as mentioned in the comment, I don't think `quota` is a param.

Reply:
Yes, correct, we are only checking with the request body only. I missed updating this comment.